quizapp/src/main/java/billennium/quizapp/service/QuizService.java

```diff
@@ -154,4 +167,21 @@ public void stopQuiz(QuizEndDto quizEndDto) {
             checkAnswers(quiz);
         });
     }
+
+    public Long addQuiz(QuizToSaveDto quizToSaveDto) {
+        QuizDefinition quizDefinition = quizDefinitionRepository.save(QuizDefinition.builder()
+                .title(quizToSaveDto.getTitle())
+                .questions(mapQuestions(quizToSaveDto.getQuestions())).build()
+        );
+
+        return quizDefinition.getId();
+    }
+
+    private List<Question> mapQuestions(List<QuestionBaseDto> questions) {
+        List<Question> questionList = new ArrayList<>();
```

Comment:
tutaj tez wydaje mi się ze przypisywanie do nowej kolekcji jest niepotrzebne. da sie to zrobić ładniej ?